inductiva/tests/tasks/test_run_task.py

```diff
@@ -0,0 +1,51 @@
+"""Test file for the run_simulation file."""
+import os
+import json
+
+import pathlib
+
+from inductiva import tasks
+
+LOGGED_TASK_ID = "1234"
+NOT_LOGGED_TASK_ID = "4321"
+
+
+def _id_in_metadata_file(task_id):
+    """Checks if the ID exists in the file."""
+    with open(tasks.TASK_METADATA_FILENAME, "r", encoding="utf-8") as f:
+        json_list = [json.loads(line) for line in f]
+    return task_id in map(lambda x: x["task_id"], json_list)
+
+
+def test_run_simulation_logging(mocker):
+    """Tests if the id of the task was added to the file."""
+
+    # Old logs may cause problems so we remove the file.
+    if os.path.exists(tasks.TASK_METADATA_FILENAME):
+        os.remove(tasks.TASK_METADATA_FILENAME)
+
+    invoke_api_mock = mocker.patch("inductiva.api.methods.invoke_async_api")
+    mocker.patch("inductiva.tasks.Task")
+
+    input_dir = pathlib.Path().cwd()
+
+    _run_simulation_and_assert("some_api_method",
+                               input_dir,
+                               invoke_api_mock,
+                               LOGGED_TASK_ID,
+                               disable_logging=False,
+                               id_exists_in_file=True)
+    _run_simulation_and_assert("some_api_method",
+                               input_dir,
+                               invoke_api_mock,
+                               NOT_LOGGED_TASK_ID,
+                               disable_logging=True,
+                               id_exists_in_file=False)
+
+
+def _run_simulation_and_assert(api_method_name, input_dir, invoke_api_mock,
```

Comment:
Instead of creating an helper function for this purpose, we should parameterize the test for the different possibilities. In this way, some of the variables are set in during the test method and not passed as arguments, while the ones that are, are really the variations we want to test. Check the `test_file_manager.py` for inspiration on that.

Reply:
yes, one test with mutiple parameters.